eng/versioning/external_dependencies.txt

```diff
@@ -193,12 +193,12 @@ com.microsoft.azure:azure-storage;8.0.0
 com.microsoft.azure:msal4j;1.13.8
 com.microsoft.azure:msal4j-persistence-extension;1.2.0
 com.sun.activation:jakarta.activation;1.2.2
-io.opentelemetry:opentelemetry-api;1.27.0
-io.opentelemetry:opentelemetry-sdk;1.27.0
-io.opentelemetry:opentelemetry-exporter-logging;1.27.0
-io.opentelemetry:opentelemetry-exporter-jaeger;1.27.0
-io.opentelemetry:opentelemetry-exporter-otlp;1.27.0
-io.opentelemetry:opentelemetry-sdk-testing;1.27.0
+io.opentelemetry:opentelemetry-api;1.28.0
+io.opentelemetry:opentelemetry-sdk;1.28.0
+io.opentelemetry:opentelemetry-exporter-logging;1.28.0
+io.opentelemetry:opentelemetry-exporter-jaeger;1.28.0
+io.opentelemetry:opentelemetry-exporter-otlp;1.28.0
+io.opentelemetry:opentelemetry-sdk-testing;1.28.0
 io.opentelemetry:opentelemetry-sdk-extension-autoconfigure;1.24.0-alpha
```

Comment:
is there any reason not to update other otel dependencies below?

Reply:
i should. autoconfigure becomes GA in 1.28.0.. Trask didn't update it in 1.27.0 update, I was planning to talk to him first.. but he is OOF today. I am going to update it anyway.